### PR TITLE
feat(ui): implement pagination of members table

### PR DESCRIPTION
**Pull request by liangfung**
Relates #1038 

1. prev/next pagination
2. page size: 20

### CODE REVIEW

**Comment by liangfung** [comment]
https://jam.dev/c/dbfa8e76-33f5-4bfa-ba74-f1459977dfe7

**Comment by wsxiaoys** [comment]
can we use standard shadcn pagination? (you don't have to use its page number part, but next/previous button at least).

https://ui.shadcn.com/docs/components/pagination

**Comment by wsxiaoys** [comment]
commented in https://jam.dev/c/884bd35b-4eac-41be-80fe-af5d2d779cde

File: ee/tabby-ui/app/(dashboard)/team/components/invitation-table.tsx [hunk 1]
```diff
@@ -58,29 +70,118 @@ const deleteInvitationMutation = graphql(/* GraphQL */ `
   }
 `)
 
+const PAGE_SIZE = 5
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/team/components/invitation-table.tsx:73]:
```suggestion
const PAGE_SIZE = 20
```

File: ee/tabby-ui/app/(dashboard)/team/components/user-table.tsx [hunk 2]
```diff
@@ -60,19 +68,34 @@ const updateUserActiveMutation = graphql(/* GraphQL */ `
   }
 `)
 
+const PAGE_SIZE = 5
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/team/components/user-table.tsx:71]:
```suggestion
const PAGE_SIZE = 20
```

**Pull request merged**